draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.spec.tsx

```diff
@@ -245,6 +245,49 @@ describe("<TitleBlockZen />", () => {
     })
   })
 
+  // TODO: fix up these tests and add coverage across the other actions
```

Comment:
I'd also suggest mocking the window dimensions? Given how buttons become different components under mobile conditions, I'd add a test for each button-like prop of the TitleBlock (primary, secondary, secondaryOverflow, etc), for each render-like prop of the button (`href`, `onClick`, `component`), for each dimension size that changes how it render  (large and small, maybe medium?).